### DISCUSSION TITLE
Welcome to GitFyle.Core.Api Discussions!

### DISCUSSION THREAD

**Discussion by cjdutoit** [The-Standard-Organization/GitFyle.Core.Api#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kandarpgautam** [comment]
I am a passionate full-stack .NET developer based in Noida, India. My zeal for technology touches all its domains. 
For the last three years, I have been involved in diverse projects with two startups. 
I am currently working both as an apprentice and a freelancer alongside an experienced senior engineer.

During my free time, I like playing guitar, and chess, blogging, and producing educational content on topics that intrigue me. 
My goal is to attain financial independence through entrepreneurship, granting me the liberty to contribute significantly and positively to society via technology.

My sole purpose for interacting with this community is to contribute and enhance my skills 
and knowledge to be able to build enterprise-level solutions that are scalable and maintainable.
Along with that, To find and connect with like-minded people. 

Here, I aim to contribute via my ideas and development skills as a dotnet developer.
Meanwhile, also learning and understanding the entire processes and practices for building solutions.

[Portfolio](https://kandycan.in/) [Personal channel](https://www.youtube.com/channel/UC7j94ldIEh1oN3pOHWk-ZVg) [Tech channel](https://www.youtube.com/channel/UCdZQXl7ZxnkuY6jdQ3qnkYw) [Instagram](https://www.instagram.com/kandarp.gautam/)

**Reply by kandarpgautam** [reply]
Here's something I was thinking about when I first heard about this idea. 
Would we wanna do something like making the produced information (texts and graphs), more portable? 
What I mean by that is, Being able to export this information in the form of markdown, 
which can also be added to repos with names such as `contributions.md` including all or top 10, etc contributors.

In this way, the repository contains the information about contributors just like a repo also contains `readme.md`
What we can do is, whenever we merge a PR, It triggers certain logic server side allowing addition or updation of `contributions.md`. This will be like a repository acknowledging the contributions of developers at a certain point in time or revisions.

Though Is this discussion specifically related to the overall project end-to-end?
